docs/integrations/destinations/s3.md

```diff
@@ -544,6 +544,7 @@ To see connector limitations, or troubleshoot your S3 connector, see more [in ou
 
 | Version     | Date       | Pull Request                                               | Subject                                                                                                                                              |
 |:------------|:-----------|:-----------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1.8.3       | 2025-05-14 | [59721](https://github.com/airbytehq/airbyte/pull/59721)   | Legacy File Transfer Uses New CDK Interface                                                                                                          |
```

Comment:
Nit: subject is copy pasted

Reply:
yeap, I have to open the PR first to get the right PR number. Updated.